app/models/team/personal.rb

```diff
@@ -27,4 +27,12 @@ def create(owner)
       team
     end
   end
+
+  def unread_count_transferred_files
+    return nil if folders.where(name: 'inbox').empty?
+
+    # rubocop:disable Metrics/LineLength
+    folders.where(name: 'inbox').first.encryptables.all.where.not(encrypted_transfer_password: nil).count
```

Comment:
we do have a boolean on File to get inbox folder, don't we?

don't ignore rubocop here ... use line breaks :wink: 

maybe add getter on personal team for inbox folder. 

also consider a getter on folder to get unread count. then you just might call it here.